staging/src/k8s.io/kube-aggregator/pkg/apiserver/handler_proxy.go

```diff
@@ -82,6 +87,10 @@ type proxyHandlingInfo struct {
 	serviceAvailable bool
 	// servicePort is the port of the service this handler proxies to
 	servicePort int32
+	// defaultSNI is the default server name to use for TLS Server Name Indication (SNI)
+	defaultSNI string
```

Comment:
Based on the "fallback" word below, I can't tell based on names if this value is used first or second.  is this a `PreferredServerName` or `AlternativeServerName`.  Please name appropriately.

Reply:
@deads2k i think preferred vs alternate, because that changes depending on the type of `Service` attached to the `APIService`. i updated this to `externalServerName`. does that seem ok?